### ISSUE TITLE
Exported PNGs and Pixen canvas using non-standard color space

### ISSUE THREAD

**Issue by Pepin-** [Pixen/Pixen#292]
I am a huge fan of Pixen, but some color space issues are preventing me from using it as much as I'd like.
From what I can tell (I'm far from an expert on this), it's doing the following things:

1. Its canvas uses some non-sRGB color space.

2. Imported png files with other color profiles are assigned the color profile matching the canvas (not converted), so they tend to look wrong. This image with an sRGB profile for instance looks lighter in Pixen vs Preview (http://goo.gl/yZPtBe). Imported png files with no color profile look wrong as well since they usually contain sRGB color data.

3. Pixen exports PNG files as Generic RGB with embedded profile, so the exported files look wrong in apps that don't respect color profiles (many don't and instead assume sRGB). Because Pixen doesn't export sRGB, the images also aren't consistent across devices.

4. RGB values set in the color picker don't consistently match the values actually painted (See: http://postimg.org/image/i1kw4hc93/). No profile choice in the drop down menu fixes this, though some are closer than others. But this might be unrelated to the other issues and might be a problem (?) with the OS X color picker rather than Pixen

I think the Pixen workspace should use sRGB or at the very least convert png files to sRGB when exporting. I suspect the app is mostly used for on-screen graphics, so it only makes sense.

**Comment by mustii82** [comment]
thats so true please implement sRgb the current color profile make so much problems :(

**Comment by QuincyMorgan** [comment]
I am seeing similar color issues, most closely related to Pepin's third point. I don't know about the technical details, but I tried this comparison:
<img width="687" alt="screen shot 2015-10-04 at 10 04 59 pm" src="https://cloud.githubusercontent.com/assets/2046746/10271789/02690fe0-6ae4-11e5-961e-a81dea596fc9.png">
iOS displayed the .png exported by Pixen incorrectly. When I used a screenshot of the Pixen image from the editing window instead, the colors came out perfectly.

I know it's probably more difficult to develop than it sounds, but having some kind of color profile management in a graphics creation application is a true necessity. I would hate to have to switch to a different app just because Pixen can't handle colors. It's one of my favorite apps!
